examples/writer.js

```diff
@@ -4,20 +4,27 @@
 
 const lib = require('../lib')
 const type = lib.type
+const timeUnit = lib.type
```

Comment:
Btw sometimes terse is nice, e.g. `const { type, timeUnit } = require('../lib')`.